kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProtocolFilterStatusFactory.java

```diff
@@ -57,9 +57,15 @@ KafkaProtocolFilter newFalseConditionStatusPatch(KafkaProtocolFilter observedPro
 
     @Override
     KafkaProtocolFilter newTrueConditionStatusPatch(KafkaProtocolFilter observedProxy,
-                                                    Condition.Type type) {
+                                                    Condition.Type type, String checksum) {
         Condition trueCondition = newTrueCondition(observedProxy, type);
         return filterStatusPatch(observedProxy, trueCondition);
     }
 
+    @Override
+    KafkaProtocolFilter newTrueConditionStatusPatch(KafkaProtocolFilter observedProxy,
+                                                    Condition.Type type) {
+        return newTrueConditionStatusPatch(observedProxy, type, "");
+    }
```

Comment:
Not a huge fan of this bit of the change however it felt better than sprinkling `, "")` across the codebase.

Looking at the diff I'm not sure it was actually better.

Reply:
Maybe define a constant `NO_CHECKSUM`. Likewise I've though of defining a `CORE_API_GROUP` for those places where we're passing a `""` to things expecting an Kube API group.